Library/Homebrew/commands.rb

```diff
@@ -194,7 +196,35 @@ def rebuild_commands_completion_list
     # Ensure that the cache exists so we can build the commands list
     HOMEBREW_CACHE.mkpath
 
+    cmds = commands(aliases: true).reject do |cmd|
+      # TODO: remove the cask check when `brew cask` is removed
```

Comment:
@MikeMcQuaid what do you think about using comments like `# TODO (2.8):` for things that should be done on `brew` minor version updates?

Reply:
This is probably a good idea. Otherwise, it's a ton to keep track of. We may as well our part to help Mike out if he's the one doing the deprecations 😉

Reply:
We already have a bunch of these in the codebase (grep for '# TODO'), so it's definitely a good idea.

Reply:
the easiest thing that gets me to find them on minor updates is a commented-out `odeprecated` or `odisabled` (because that's what I search for).